exercises/10/readme.md

```diff
@@ -11,6 +11,7 @@ At the end of these steps, your project will be deployed to SAP Cloud Platform C
 ### 1. Sign in to Cloud Foundry
 
 :point_right: Run the following command from your command line to log in to the referenced Cloud Foundry endpoint. When prompted use your SAP Cloud Platform credentials
```

Comment:
- please don't forget period at end of sentence

Reply:
done!